### DISCUSSION TITLE
How to mv multiple files at once?

### DISCUSSION THREAD

**Discussion by sam701** [nushell/nushell#6091] [Q&A] [answered]
How would one do the same in nushell?
```
mv abc def ghi /tmp/test/
```

**Comment by fdncred** [comment, marked as answer]
here's one way but it seems harder than it should be 
```
[abc def ghi] | each {|f| mv $f t2}
```

**Reply by sam701** [reply]
@fdncred thanks! Are there any plans or activities to simplify similar tasks?

**Reply by fdncred** [reply]
We'd accept a PR to help with this. We're largely supported by contributors for functionality like this.

**Reply by PlasmaIntec** [reply]
Working on this in #6103

**Reply by sam701** [reply]
@PlasmaIntec great news! I added a comment to your PR. 

Thank you for addressing this issue so fast.
